inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -125,7 +125,7 @@ def __init__(self,
             # If an exception occurs during string conversion, raise that exception
             raise SotaError('package_list is not a string in manifest') from e
 
-        self._device_reboot = parsed_manifest['deviceReboot']
+        self._reboot_device = parsed_manifest['deviceReboot']        
```

Comment:
**Suggestion:** The self._reboot_device attribute is set directly from the parsed_manifest without validation. It's assumed to be a string that contains some form of affirmative or negative response. Consider adding validation to ensure that the value is within the expected set of values to avoid potential bugs or unexpected behavior if the manifest contains unexpected values. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._reboot_device = parsed_manifest.get('deviceReboot', 'Yes').strip().lower()
        if self._reboot_device not in ["yes", "no"]:
            raise ValueError("Invalid value for 'deviceReboot' in manifest")
```